packages/common-integration-tests/src/auth-client.ts

```diff
@@ -4,8 +4,11 @@ import {
   MomentoErrorCode,
   RefreshAuthToken,
 } from '@gomomento/sdk-core';
-import {IAuthClient} from '@gomomento/sdk-core/dist/src/internal/clients/auth/IAuthClient';
+import {IAuthClient} from '@gomomento/sdk-core/dist/src/clients/IAuthClient';
 import {expectWithMessage} from './common-int-test-utils';
+import {Restrictions} from '@gomomento/sdk-core/dist/src/auth/tokens/token-scope';
+
+const SUPER_USER_RESTRICTIONS: Restrictions = {restrictions: []};
```

Comment:
Here's how I can express SuperUser restrictions for use cases like the console, without exposing directly to users.